README.md

```diff
@@ -58,6 +58,32 @@ Before opening a PR, check the service renders successfully at http://localhost:
 and run all the tests locally as described below.
 
 ## Adding to your service
+
+### Users of the latest versions of play-ui or play-frontend-hmrc
+
+If you are using the latest version of [hmrc/play-ui](https://github.com/hmrc/play-ui#accessibility-statements)
```

Comment:
Maybe state minimum versions of `play-ui` and `play-frontend-hmrc` for this to work?

Reply:
Yes, good idea.